api/profile/models/profile.settings.json

```diff
@@ -22,9 +22,9 @@
       "type": "string"
     },
     "user": {
+      "via": "profile",
```

Comment:
can you reset `profile.settings.json` from the commit since it didn't change, same for `courses.settings.json`